src/AcceptanceTests/Receiving/When_receiving_a_message.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace NServiceBus.Transport.AzureServiceBus.AcceptanceTests.Sending.Receiving
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NUnit.Framework;
+    using Pipeline;
+
+    public class When_receiving_a_message
+    {
+        [Test]
+        public async Task Should_have_access_to_the_original_brokered_message_via_the_context_bag()
+        {
+            await Scenario.Define<Context>()
+                .WithEndpoint<Endpoint>(b => b.When(
+                    (session, c) => session.SendLocal(new Message())))
+                .Done(c => c.LockedUntilUtcFromHandler == c.LockedUntilUtcFromBehavior)
```

Comment:
Should the test still be looking this property when we just care about the message type being there?

Reply:
Agreed. Updated.